Source/AquaMira.F7/Apps/MeadowProjectLabApp.cs

```diff
@@ -30,6 +30,31 @@ public override Task Initialize()
         }, TaskCreationOptions.LongRunning)
             .Start();
 
+        var svc = Resolver.UpdateService;
+
+        svc.UpdateAvailable += async (updateService, info, token) =>
+        {
+            Resolver.Log.Info($"Update available!", "AquaMira");
+        };
+
+        svc.UpdateRetrieved += async (updateService, info, token) =>
+        {
+            Resolver.Log.Info($"Update retrieved!", "AquaMira");
+        };
+
+        svc.UpdateFailure += async (updateService, info, token) =>
+        {
+            Resolver.Log.Error($"Update failed: {info.Name}", "AquaMira");
+        };
+
+        svc.RetrieveProgress += async (updateService, info, token) =>
+        {
+            Resolver.Log.Info(
+                $"Update progress: {info.DownloadProgress} bytes downloaded",
```

Comment:
this will get super chatty if we're sending to the cloud, no?

Reply:
Yeah, probably. We can save this for when we have bulk events fully implemented.